avocado/core/nrunner.py

```diff
@@ -750,6 +766,9 @@ class BaseRunnerApp:
         (('-u', '--uri'),
          {'type': str, 'default': None, 'help': 'URI of runnable'}),
 
+        (('-c', '--config'),
+         {'type': str, 'default': '{}', 'help': 'A config json data.'}),
```

Comment:
This is user visible, so let's use `JSON` in the help message.  Also, other messages don't include a trailing dot.